### PR TITLE
java codegen: integration test for LF 1.11

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by stefanobaghino-da — APPROVED** [review]
LGTM.

The formatting looks a bit funky here and there, we don't enforce a style but probably running the IntelliJ formatter before merging would be a good idea to minimize diff noise in the future.

File: language-support/java/codegen/BUILD.bazel [hunk 1]
```diff
@@ -158,20 +150,13 @@ test_daml_lf_target_versions = [
             ],
         ),
     ]
-    for target in test_daml_lf_target_versions
+    for target in COMPILER_LF_VERSIONS
```
Comment (cocreature) [inline language-support/java/codegen/BUILD.bazel:153]: :+1:

**Review by cocreature — COMMENTED** [review]
Thanks Remy!

**Pull request merged**